starter-code/routes/index.js

```diff
@@ -0,0 +1,46 @@
+const express = require("express");
+const router = express.Router();
+
+const User = require("../user/user.js");
+
+const bcrypt         = require("bcrypt");
+const bcryptSalt     = 10;
+
+
+//GET home page.
+router.get("/", function(req, res, next){
+  res.render("index");
+});
+
+router.post("/", (req, res, next) => {
+  console.log(req.body);
+  let username = req.body.username;
+  let password = req.body.password;
+  let salt     = bcrypt.genSaltSync(bcryptSalt);
+  let hashPass = bcrypt.hashSync(password, salt);
+
+  if (username === "" || password === "") {
```

Comment:
inside two of these `if (...)` it is just logging but not returning a response to the browser

it should also `res.render(...name of the signup template...)` with a specific error message